src/motion/utils/use-visual-element.ts

```diff
@@ -27,7 +20,6 @@ type VisualElementRef = MutableRefObject<VisualElement | null>
 
 export function useVisualElement<E>(
     createVisualElement: CreateVisualElement<E>,
-    Component: string | ComponentType,
```

Comment:
I don't really understand how we can remove this Component argument now?

Reply:
This is now getting provided to these functions when we build the component, rather than threading it through with every individual render.

Reply:
Specifically with this function, it's getting bound to the provided `createVisualElement` function, which is nice as this is meant to be the abstracted function that deals with platform-specific (in this case DOM) functionality